tests/test_cli.py

```diff
@@ -370,6 +370,35 @@ def test_export_python_default_filepath(self):
 
         self.assertEqual(exit_cm.exception.code, 0)
 
+    def test_export_python_with_datastack(self):
+        """CLI: Export a python script including args, for a given model."""
+        from natcap.invest import cli
+
+        datastack_dict = {
+            'model_name': 'natcap.invest.carbon',
+            'invest_version': '3.10',
+            'args': {}
+        }
+        new_parameter_set_path = os.path.join(
+            self.workspace_dir, 'paramset.invs.json')
+        with open(new_parameter_set_path, 'w') as parameter_set_file:
+            parameter_set_file.write(
+                json.dumps(datastack_dict, indent=4, sort_keys=True))
+
+        target_filepath = os.path.join(self.workspace_dir, 'foo.py')
+        with redirect_stdout() as stdout_stream:

```

Comment:
My understanding is this just prevents the stdout from showing up in the test output? If `stdout_stream` is not used, can this be simplified?
```suggestion
        with redirect_stdout():
```
```